config/rbac/knative/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+# ---------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ---------------------------------------------------------------------------
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: my-ns
+resources:
+# Required in order to perform namespace replacement below
+- operator-service-account.yaml
+- operator-role-knative.yaml
+- operator-role-binding-knative.yaml
+- operator-cluster-role-binding-custom-resource-definitions.yaml
+replacements:
```

Comment:
@gansheer I think the "replacement" can be a good way to solve the plethora of "placeholder" replacement we have in kustomization. The idea is that we include the serviceAccount (altough it exists) in order to get its namespace and replace it accordingly. wdyt?

Reply:
Using `replacements` is one of my (favorite) possible option for namespace handling. Bu I need to see how it fits with all the other rbac configurations adaptations like ClusterRoleBindings and RoleBindings that reference the service.